packages/material-ui-codemod/src/v5.0.0/jss-to-styled.test/withCreateStyles1.expected.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { styled } from '@material-ui/core/styles';
+const PREFIX = 'withCreateStyles1';
+
+const classes = {
+  root: `${PREFIX}-root`
+};
+
+const Root = styled('div')((
+  {
+    theme
+  }
+) => ({
+  [`&.${classes.root}`]: {
+    background: theme.background,
+  }
+}));
+
+const MyComponent = (props) => {
+  const { } = props;
```

Comment:
Will this break the local development? I hope it is not.

Reply:
I don't think so. I noticed it on the store, but to be honest I disregard it in order to focus on more important things first.